app/views/admin/products_v3/_filters.html.haml

```diff
@@ -1,6 +1,7 @@
 = form_with url: admin_products_path, id: "filters", method: :get, data: { "search-target": "form", 'turbo-frame': "_self" } do
   = hidden_field_tag :page, nil, class: "page"
   = hidden_field_tag :per_page, nil, class: "per-page"
+  = hidden_field_tag '[q][s]', params.dig(:q, :s) || 'name asc', class: 'sort', 'data-default': 'name asc'
```

Comment:
Hmm we're now sharing the same filters code with the orders page, I wonder if we can DRY it into a shared partial or component. Next time.. 🤔

Reply:
Oh yes, I had that in mind but forgot while looking for the disconnected form issue 😅.

Reply:
No worries, I totally understand the distraction! I think it's fine for now.